### PR TITLE
Add uninstall command

### PR DESCRIPTION
**Pull request by iamhsa**
Hi,
To have CRUD functions I added uninstall command.
Have a nice day.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,11 @@ $ tfenv use latest
 $ tfenv use latest:^0.8
 ```
 
+### pkenv uninstall
```
Comment (kamatama41) [inline README.md]: Please replace pkenv and Packer to tfenv and Terraform

File: test/test_uninstall.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+echo "### Uninstall local versions"
+cleanup || error_and_die "Cleanup failed?!"
+
+for v in 0.1.0; do
+  tfenv install ${v} || error_and_proceed "Install of version ${v} failed"
+done
+
+tfenv uninstall 0.1.0 || error_and_proceed "Uninstall of version ${v} failed"
+
+#tfenv list | grep 0.1.0 && exit 1 || exit 0
```
Comment (kamatama41) [inline test/test_uninstall.sh]: Why this is commented? 
I think that you should check the uninstalled version does not appear in the list command.

**Comment by iamhsa** [comment]
My bad ! I fixed mistakes.

File: test/test_uninstall.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+echo "### Uninstall local versions"
+cleanup || error_and_die "Cleanup failed?!"
+
+for v in 0.1.0; do
+  tfenv install ${v} || error_and_proceed "Install of version ${v} failed"
+done
+
+tfenv uninstall 0.1.0 || error_and_proceed "Uninstall of version ${v} failed"
+
+tfenv list | grep 0.1.0 && exit 1 || exit 0
```
Comment (kamatama41) [inline test/test_uninstall.sh]: Please rewrite the test case `### Uninstall local versions` (L22~28) to the same approach of other tests such as the following.

```sh
v=0.1.0
(
  tfenv install ${v} || exit 1
  tfenv uninstall ${v} || exit 1
  tfenv list | grep ${v} || exit 1
) || error_and_proceed "Uninstall of version ${v} failed"
```

Comment (iamhsa) [inline test/test_uninstall.sh]: Done ! Should I add some test for uninstall latest, uninstall latest:^x.x.x ?

**Comment by kamatama41** [comment]
LGTM 👍 
Thank you for your contribution 😃

File: test/test_uninstall.sh (hunk 3 above)
Comment (kamatama41) [inline test/test_uninstall.sh]: @iamhsa 
Sorry, I overlooked your reply.
Yes, it's better to add that test case.

**Pull request merged**